pkg/torcx/profile_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 CoreOS Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package torcx
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+)
+
+// format of test-get-profile.json:
+//
+// {
+//   "kind": "profile-manifest-v0",
+//   "value": {
+//     "images": [
+//       {
+//         "name": "test-name",
+//         "reference": "test-reference"
+//       }
+//     ]
+//   }
+// }
+
+func TestGetProfile(t *testing.T) {
+	profilePath := "../../fixtures/test-get-profile.json"
```

Comment:
I would probably remove the comment above and just insert a "v0" in the file name. If you want, you can linke the md file from schema here.